### ISSUE TITLE
Bower.json does not follow the spec

### ISSUE THREAD

**Issue by MikeParkin** [gregjacobs/Autolinker.js#226]
Hello, 

I've got an issue in my application (oro platform) pulling down Autolinker.js as a dependency - as it uses an uppercase name in bower.json. 

The spec for bower says that this should be lowercase, not uppercase: https://github.com/bower/spec/blob/master/json.md

Would you consider fixing this here, so our application works correctly? We are using the 1.4.x branch, so I'm hoping you can fix this in 1.4.5?

```
name

Required

Type: String

The name of the package as stored in the registry.

Must be unique.
Should be slug style for simplicity, consistency and compatibility. Example: unicorn-cake
Lowercase, a-z, can contain digits, 0-9, can contain dash or dot but not start/end with them.
Consecutive dashes or dots not allowed.
50 characters or less.
```

**Comment by gregjacobs** [comment]
Hey @MikeParkin. So I just saw this issue. Are you still using Bower? 

I'm working on a 2.0 version and I was considering dropping Bower altogether since npm seems to be the clear winner even for web packages these days

**Comment by gregjacobs** [comment]
Fixed this in v3.0.3
